src/js/App/Sidenav/__snapshots__/SectionNav.test.js.snap

```diff
@@ -52,13 +52,13 @@ exports[`SectionNav should render app with subapp and custom url, but without gr
           data-ouia-component-type="PF4/NavItem"
           data-ouia-safe="true"
         >
-          <a
+          <div
```

Comment:
It's really strange that all `a` elements were replaced with `div`

Reply:
Yeah this should be double checked. There should still be a `a` tag in here